api/model/api.go

```diff
@@ -1200,6 +1200,8 @@ type GraphQLResponseExtensions struct {
 }
 
 type GraphQLProxyConfig struct {
+	// +nullable
+	Features GraphQLProxyFeaturesConfig `json:"features"`
```

Comment:
@pvormste will this feature be required for users to provide? If not, we can make it optional as follows:
```suggestion
	Features GraphQLProxyFeaturesConfig `json:"features,omitempty"`
```